### PR TITLE
Base camp expansion storehouse wood (1/8)

### PR DESCRIPTION
**Pull request by PatrikLundell**
#### Summary

Content "Base camp storehouse 2 expansion in wood with wooden racks only"

#### Purpose of change

Provide a storehouse expansion version that maximizes storage capability with a range of storage furniture.

#### Describe the solution

This expansion rotates/mirrors depending on which expansion slot it's placed in to provide vehicle access to the hub from each cardinal direction if placed adjacent to another expansion from the version 2 family.

The storehouse itself is a large building that just houses storage furniture, and it comes in two sizes: the "normal" half tile size and the full one. One reason for the half size version is that when placed in a cardinal direction expansion (N/E/S/W) every tile of the storehouse remains within the reality bubble as long as the player is in any tile of the base, including expansions. When placed in one of the four diagonal expansions, this is true only for a quarter of the tiles.

This PR introduces the expansion version itself and supports a single building material, namely wooden panels. Similarly, it introduces a single storage furniture type (wooden racks). Follow on PRs will introduce additional building materials as well as additional storage furniture types (8 building materials and 6 furniture types in total).

There are two main reasons for providing different alternatives for the storage furniture:
- Material availability. By providing different types of furniture the player can elect to build furniture that uses construction materials that are most easily available.
- Organization/role playing. By building different storage sections out of different materials the player can match the type of items stored with the type of furniture to store it in. This can also be done to make different storage zones distinguishable by using different furniture in different zones.

Each type of furniture is described with three parameters the author deemed important, namely storage capacity, whether it hides its contents, and whether it blocks LoS. A clear disadvantage of this approach is that changes to these properties of the furniture would have to be carried over to the recipes manually, and there are no means to automatically detect when/if this happens.

Note that there is no connection between furniture type and building material, so the fact that they are introduced as apparent pairs is just a matter of convenience. Since there are more building materials than furniture variants the "left over" building material PRs will use mixed furniture in the images showing the tested construction.

While the expansion ought to be placed in one of the cardinal expansion slots, this PR places it in the NE slot simply because the expansion should still work in all slots, and I've started the other expansion versions out at that position, going though the others in following PRs, and haven't seen a compelling reason to change that process.

#### Describe alternatives you've considered

I tried to implement a succession of three building sizes rather than two, but it got awkward with the middle size placing a wall that would block off the full size version.

#### Testing

It was tested when originally developed. It's now tested by implementing the expansion in one expansion slot (NE), to ensure the rotation/mirroring flags are added properly to all recipes. The base configuration is the NW version, so any failure to introduce the set of flags will result in things being placed in the wrong location.

#### Additional context

The "normal" sized storehouse.
Screenshot (56)

The hoarder version.
Screenshot (57)

### CODE REVIEW

**Comment by PatrikLundell** [comment]
I need help to deal with the incomprehensible (to me) error generated by Text Changes Analyzer / analyze-text-changes (pull_request).

As far as I can tell, it complains that a file that's in an accepted PR conflicts with the same file in this PR which is directly based on the previous PR (the branch was made from a branch that was created from a branch that contained that PR). Updating to the current master does not resolve it, and the "hint" command doesn't work ("error: option 'show-current-patch' takes no value", and without the "=diff" part I get "fatal: Resolve operation not in progress, we are not resuming"). I haven't tried the "--abort" option as I have no idea what "am" is supposed to be doing, and "abort" seems destructive.

**Comment by kevingranade** [comment]
tl;dr don't worry about it.

I believe what's happening here is that the workflow can't handle the fact that there are (trivial, ignorable) conflicts when merging this PR due to the way the branches involved are structured.  This is certainly not a problem as your PR doesn't introduce any translatable strings, and git/github have no problems resolving the conflicts with a squash merge, which is what I've been doing with all of these PRs.

@BrettDong may or may not want to look into this failure mode, but it's not critical as we don't normally see branching patterns like this.
